### ISSUE TITLE
Cannot turn off Rails/ActiveRecordAliases cop

### ISSUE THREAD

**Issue by huyvohcmc** [bbatsov/rubocop#5677]
*Be clear, concise and precise in your description of the problem.
Open an issue with a descriptive title and a summary in grammatically correct,
complete sentences.*

*Use the template below when reporting bugs. Please, make sure that
you're running the latest stable RuboCop and that the problem you're reporting
hasn't been reported (and potentially fixed) already.*

*Before filing the ticket you should replace all text above the horizontal
rule with your own words.*

--------

## Expected behavior

Rubocop ignore all Rails/ActiveRecordAliases warnings

## Actual behavior

It still shows warnings

## Steps to reproduce the problem

In .rubocop.yml

```
Rails:
  Enabled: true

Rails/ActiveRecordAliases:
  Enabled: false
```

## RuboCop version

```
$ rubocop -V
0.53.0 (using Parser 2.5.0.3, running on ruby 2.3.3 x86_64-darwin17)
```

**Comment by pocke** [comment]
I cannot reproduce this problem.

# Steps to reproduce the problem (I cannot reproduce it.)

Firstly, put `test.rb`


```ruby
User.update_attributes(foo: bar)
```

Next, put `.rubocop.yml` (without disabling the cop)

```yaml
Rails:
  Enabled: true
```

Then, run `rubocop`. It displays an offense for the cop.

```bash
$ rubocop 
Inspecting 1 file
W

Offenses:

test.rb:1:6: W: Rails/ActiveRecordAliases: Use update instead of update_attributes.
User.update_attributes(foo: bar)
     ^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

Next, update the `.rubocop.yml` to disable the cop.

```yaml
Rails:
  Enabled: true

Rails/ActiveRecordAliases:
  Enabled: false
```

Then, run `rubocop`. It does not display the offense. But in your case, it displays the offense.

```bash
$ rubocop 
Inspecting 1 file
.

1 file inspected, no offenses detected
```


---

Can you check your `.rubocop.yml` has exactly one `Rails/ActiveRecordAliases`? Sometimes we write configurations for the same cop twice.

```yaml
Rails:
  Enabled: true

Rails/ActiveRecordAliases:
  Enabled: false

```

**Comment by huyvohcmc** [comment]
I edited the wrong file, so sorry for this.
